tests/ui_tests/test_jobs_page.py

```diff
@@ -0,0 +1,197 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Union
+
+import os
+import pytest
+import allure
+
+from adcm_client.objects import ADCMClient, Cluster, Bundle, Provider, Service, Host
+from adcm_pytest_plugin import utils
+from adcm_pytest_plugin.steps.actions import run_cluster_action_and_assert_result
+
+from tests.ui_tests.app.app import ADCMTest
+from tests.ui_tests.app.page.job_list.page import (
+    JobListPage,
+    JobStatus,
+    TableTaskInfo,
+    TaskInfo,
+)
+
+
+LONG_ACTION_DISPLAY_NAME = 'Long action'
+SUCCESS_ACTION_DISPLAY_NAME = 'Success action'
+FAIL_ACTION_DISPLAY_NAME = 'Fail action'
+
+CLUSTER_NAME = 'Great Cluster'
+SERVICE_NAME = 'Serviceno'
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture()
+def page(app_fs: ADCMTest, login_to_adcm_over_api) -> JobListPage:
+    return JobListPage(app_fs.driver, app_fs.adcm.url).open()
+
+
+@allure.title("Upload cluster bundle")
+@pytest.fixture()
+def cluster_bundle(sdk_client_fs: ADCMClient) -> Bundle:
+    return sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), "cluster"))
+
+
+@allure.title("Upload provider bundle")
+@pytest.fixture()
+def provider_bundle(sdk_client_fs: ADCMClient) -> Bundle:
+    return sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), "provider"))
+
+
+@allure.title("Create cluster")
+@pytest.fixture()
+def cluster(cluster_bundle: Bundle) -> Cluster:
```

Comment:
we use it in few files, maybe to replace it to common or some other place?

Reply:
They depend on the current file (to get data directory), if we'll move it to conftest, then we'll need to pass this as param (more parametrization above tests) or inspect caller. I'll check if it is an easy thing to do and if it is I'll try to move it to conftest.